src/main/java/com/sapred/ordermanagerred/service/ProductCategoryService.java

```diff
@@ -26,6 +26,23 @@ public class ProductCategoryService {
     @Autowired
     private ProductCategoryMapper productCategoryMapper;
 
+    //יצירת מופע של productCategory
+    public ResponseEntity<String> createProductCategory(ProductCategory productCategory){
+        try {
+            if(productCategoryRepository.existsByName(productCategory.getName()))
+                throw new ResponseStatusException(HttpStatus.CONFLICT,"the name of the category already exists");
+            AuditData auditData=new AuditData();
+            auditData.setCreateDate(new Date());
+            auditData.setUpdateDate(new Date());
+            productCategory.setAuditData(auditData);
+            productCategoryRepository.save(productCategory);
+            return ResponseEntity.ok("success: true");
```

Comment:
ResponseEntity only in controller

Reply:
It's not related to my mission, but I think it's fixed